assets/js/components/PostSearcherAutoSuggest.js

```diff
@@ -99,14 +108,20 @@ export default function PostSearcherAutoSuggest( {
 				.catch( () => setResults( [] ) )
 				.finally( () => setIsLoading?.( false ) );
 		}
-	}, [ debouncedValue, setIsLoading ] );
+	}, [ debouncedValue, setIsLoading, searchTerm ] );
 
 	useEffect( () => {
 		if ( ! searchTerm ) {
 			setResults( [] );
 		}
 	}, [ searchTerm ] );
 
+	useEffect( () => {
+		if ( currentEntityTitle ) {
+			setSearchTerm( currentEntityTitle );
+		}
+	}, [ currentEntityTitle, unifiedDashboardEnabled ] );
```

Comment:
I believe `unifiedDashboardEnabled` is not needed for this `useEffect` hook, right?
```suggestion
	}, [ currentEntityTitle ] );
```

Reply:
You're right @eugene-manuilov . Most likely some leftover when I originally had the test for unifiedDashboard. Thanks for catching that 👍